src/components/Dashboard/TodoList.vue

```diff
@@ -463,7 +465,7 @@ export default {
       deleteErrorDialog: false,
       cancelPaymentErrors: [] as Array<any>,
       cancelPaymentErrorDialog: false,
-      confirmCheckbox: false,
+      enableCheckbox: [],
```

Comment:
It feels like this array should be sized (allocated) before it's used in the template but I dunno.

Reply:
i'm not sure exactly what you mean. Couldn't this potentially be any size?

Reply:
~~I'm not 100% sure how arrays are allocated in JS.~~ I just read up on this. JS arrays are dynamically resizable.

In other languages, you'd have to specify the size (length) before using any arbitrary index (as you are doing in the template), otherwise you'd get a runtime array bounds exception or memory corruption.

Looks like we can get away with this magic after all:
https://stackoverflow.com/a/33903611

I can't quickly find the official docs but I found this:
https://www.educba.com/dynamic-array-in-javascript/